features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/NRFCordioHCIDriver.cpp

```diff
@@ -96,42 +96,54 @@ static const uint16_t maxRptBufSize = 12 + 2 + 255;
 static const uint16_t aclBufSize = 12 + connDataLen + 4 + BB_DATA_PDU_TAILROOM;
 
 const LlRtCfg_t NRFCordioHCIDriver::_ll_cfg = {
-    /* Device */
-    /*compId*/         			LL_COMP_ID_ARM,
-    /*implRev*/					LL_IMPL_REV,
-    /*btVer*/         			LL_VER_BT_CORE_SPEC_5_0,
-    0, // padding 
-    /* Advertiser */
-    /*maxAdvSets*/         		4, // 4 Extended Advertising Sets
-    /*maxAdvReports*/         	8,
-    /*maxExtAdvDataLen*/         advDataLen,
-    /*defExtAdvDataFrag*/        64,
-    0, // Aux delay
-    /* Scanner */
-    /*maxScanReqRcvdEvt*/         4,
-    /*maxExtScanDataLen*/         advDataLen,
-    /* Connection */
-    #if defined(NRF52840_XXAA)
-    /*maxConn*/          4,
-    #else
-    /*maxConn*/          2,
-    #endif
-    /*numTxBufs*/          numTxBufs,
-    /*numRxBufs*/          numRxBufs,
-    /*maxAclLen*/          connDataLen,
-    /*defTxPwrLvl*/          0,
-    /*ceJitterUsec*/          0,
-    /* DTM */
-    /*dtmRxSyncMs*/          10000,
-    /* PHY */
-    /*phy2mSup*/          TRUE,
-    #if defined(NRF52840_XXAA)
-    /*phyCodedSup*/          TRUE,
-    #else
-    /*phyCodedSup*/          FALSE,
-    #endif
-    /*stableModIdxTxSup*/         TRUE,
-    /*stableModIdxRxSup*/          TRUE
+/* Device */
```

Comment:
Would it be possible to add relevant flags to the configuration: 
- maxAdvSets
- maxAdvReports
- maxExtAdvDataLen
- defExtAdvDataFrag
- maxScanReqRcvdEvt
- maxExtScanDataLen
- maxConn
- phy2mSup
- phyCodedSup

There might be more. Also, we may want to allow version 4 of bluetooth (btVer) for applications that don't use BT 5 features.

Reply:
added config